src/components/Run.tsx

```diff
@@ -125,6 +148,10 @@ export const Run: React.FC<Props> = ({ runDoc, gameDoc }) => {
     await setDoc(runDoc.ref, { hidden: false }, { merge: true });
   };
 
+  const shoutoutTime = shoutoutTimes[run.category || ""];
+  const isShoutout =
+    shoutoutTime !== undefined && run.timeParsed < shoutoutTime;
```

Comment:
this should work too, if you want
```suggestion
  const shoutoutTime = shoutoutTimes[run.category];
  const isShoutout = run.timeParsed < shoutoutTime;
```
`shoutoutTime` will be undefined if `run.category` isn't a key in `shoutoutTimes`, and `number < undefined` is always false